website/docs/reference/resource-configs/trino-configs.md

```diff
@@ -0,0 +1,335 @@
+---
+title: "Starburst (Trino) configurations"
+id: "trino-configs"
+---
+
+
+## Model-level configuration
+
+### Session properties
+
+In some specific cases, there may be needed tuning through the Trino session properties only for a specific dbt model. In such cases, using the [dbt hooks](https://docs.getdbt.com/reference/resource-configs/pre-hook-post-hook) may come to the rescue:
+
+```sql
+{{
+  config(
+    pre_hook="set session query_max_run_time='10m'"
+  )
+}}
+```
+
+### Use table properties to configure connector specifics
+
+Trino connectors use table properties to configure connector specifics.
+
+Check the Trino connector documentation for more information.
+
+```sql
+{{
+  config(
+    materialized='table',
+    properties= {
+      "format": "'PARQUET'",
+      "partitioning": "ARRAY['bucket(id, 2)']",
+    }
+  )
+}}
+```
+
+## Materialization-specific configs
+
+### Seeds
+
+Seeds are CSV files in your dbt project (typically in your data directory), that dbt can load into your data warehouse using the dbt seed command.
+
+For dbt-trino batch_size is defined in macro `trino__get_batch_size()` and default value is `1000`.
+In order to override default value define within your project a macro like the following:
+
+```jinja2
+{% macro default__get_batch_size() %}
+  {{ return(10000) }}
+{% endmacro %}
+```
+
+### Table
+
+`dbt-trino` supports two modes in `table` materialization `rename` and `drop` configured using `on_table_exists`.
+
+- `rename` - creates intermediate table, then renames the target to backup one and renames intermediate to target one.
+- `drop` - drops and recreates a table. It overcomes table rename limitation in AWS Glue.
+
+By default `table` materialization uses `on_table_exists = 'rename'`, see an examples below how to change it.
+
+In model add:
+
+```jinja2
+{{
+  config(
+    materialized = 'table',
+    on_table_exists = 'drop`
+  )
+}}
+```
+
+or in `dbt_project.yaml`:
+
+```yaml
+models:
+  path:
+    materialized: table
+    +on_table_exists: drop
+```
+
+Using `table` materialization and `on_table_exists = 'rename'` with AWS Glue may result in below error:
+
+```sh
+TrinoUserError(type=USER_ERROR, name=NOT_SUPPORTED, message="Table rename is not yet supported by Glue service")
+```
+
+### View
+
+Adapter supports two security modes in `view` materialization `DEFINER` and `INVOKER` configured using `view_security`.
```

Comment:
```suggestion
Adapter supports two security modes in `view` materialization `definer` and `invoker` configured using `view_security`.
```

Lowercase this since it was lowercase above for tables